Translations/pt_BR.po

```diff
@@ -4289,403 +4293,403 @@ msgstr ""
 msgid "Red Death"
-msgstr ""
+msgstr "Morte Vermelha"
 
 #: Source/monstdat.cpp:88
 msgid "Litch Demon"
-msgstr ""
+msgstr "Demônio Litch"
 
 #: Source/monstdat.cpp:89
 msgid "Undead Balrog"
-msgstr ""
+msgstr "Balrog Morto-Vivo"
 
 #: Source/monstdat.cpp:90
 msgid "Incinerator"
-msgstr ""
+msgstr "Incinerador"
 
 #: Source/monstdat.cpp:91
 msgid "Flame Lord"
-msgstr ""
+msgstr "Senhor das Chamas"
 
 #: Source/monstdat.cpp:92
 msgid "Doom Fire"
-msgstr ""
+msgstr "Fogo da Perdição"
 
 #: Source/monstdat.cpp:93
 msgid "Hell Burner"
-msgstr ""
+msgstr "Queimador Infernal"
 
 #: Source/monstdat.cpp:94
 msgid "Red Storm"
-msgstr ""
+msgstr "Tempestade Vermelha"
 
 #: Source/monstdat.cpp:95
 msgid "Storm Rider"
-msgstr ""
+msgstr "Ginete da Tempestade"
 
 #: Source/monstdat.cpp:96
 msgid "Storm Lord"
-msgstr ""
+msgstr "Senhor da Tempestade"
 
 #: Source/monstdat.cpp:97
 msgid "Maelstorm"
-msgstr ""
+msgstr "Maelstorm"
 
 #: Source/monstdat.cpp:98
 msgid "Devil Kin Brute"
-msgstr ""
+msgstr "Bruto Cria do Diabo"
 
 #: Source/monstdat.cpp:99
 msgid "Winged-Demon"
-msgstr ""
+msgstr "Demônio Alado"
 
 #: Source/monstdat.cpp:100
 msgid "Gargoyle"
-msgstr ""
+msgstr "Gárgula"
 
 #: Source/monstdat.cpp:101
 msgid "Blood Claw"
-msgstr ""
+msgstr "Garra de Sangue"
 
 #: Source/monstdat.cpp:102
 msgid "Death Wing"
-msgstr ""
+msgstr "Asa da Morte"
 
 #: Source/monstdat.cpp:103
 msgid "Slayer"
-msgstr ""
+msgstr "Homicida"
 
 #: Source/monstdat.cpp:104 Source/spelldat.cpp:28
 msgid "Guardian"
 msgstr "Guardião"
 
 #: Source/monstdat.cpp:105
 msgid "Vortex Lord"
-msgstr ""
+msgstr "Senhor do Vórtice"
 
 #: Source/monstdat.cpp:106
 msgid "Balrog"
-msgstr ""
+msgstr "Balrog"
 
 #: Source/monstdat.cpp:107
 msgid "Cave Viper"
-msgstr ""
+msgstr "Víbora da Caverna"
 
 #: Source/monstdat.cpp:108
 msgid "Fire Drake"
-msgstr ""
+msgstr "Draco de Fogo"
 
 #: Source/monstdat.cpp:109
 msgid "Gold Viper"
-msgstr ""
+msgstr "Víbora de Ouro"
 
 #: Source/monstdat.cpp:110
 msgid "Azure Drake"
-msgstr ""
+msgstr "Draco Lazúli"
 
 #: Source/monstdat.cpp:111
 msgid "Black Knight"
-msgstr ""
+msgstr "Cavaleiro Negro"
 
 #: Source/monstdat.cpp:112
 msgid "Doom Guard"
-msgstr ""
+msgstr "Guarda da Perdição"
 
 #: Source/monstdat.cpp:113
 msgid "Steel Lord"
-msgstr ""
+msgstr "Senhor do Aço"
 
 #: Source/monstdat.cpp:114
 msgid "Blood Knight"
-msgstr ""
+msgstr "Cavaleiro Sangrento"
 
 #: Source/monstdat.cpp:115
 msgid "The Shredded"
-msgstr ""
+msgstr "Desfiador"
 
 #: Source/monstdat.cpp:116
 msgid "Hollow One"
-msgstr ""
+msgstr "Oco"
 
 #: Source/monstdat.cpp:117
 msgid "Pain Master"
-msgstr ""
+msgstr "Mestre da Dor"
 
 #: Source/monstdat.cpp:118
 msgid "Reality Weaver"
-msgstr ""
+msgstr "Tecelão de Realidades"
 
 #: Source/monstdat.cpp:119
 msgid "Succubus"
-msgstr ""
+msgstr "Súcubo"
 
 #: Source/monstdat.cpp:120
 msgid "Snow Witch"
-msgstr ""
+msgstr "Bruxa da Neve"
 
 #: Source/monstdat.cpp:121
 msgid "Hell Spawn"
-msgstr ""
+msgstr "Cria do Inferno"
```

Comment:
We already used "Cria" for "Kin" . I think we should consider using different words for these to differentiate.
```suggestion
msgstr "Prole do Inferno"
```